src/cdk/tree/tree.spec.ts

```diff
@@ -0,0 +1,161 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {Component, ViewChild} from '@angular/core';
+import {CdkTree} from './tree';
+import {FlatNode, NestedNode} from './tree-data';
+import {CollectionViewer, DataSource} from '@angular/cdk/collections';
+import {BehaviorSubject} from 'rxjs/BehaviorSubject';
+import {Observable} from 'rxjs/Observable';
+import {combineLatest} from 'rxjs/observable/combineLatest';
+import {CdkTreeModule} from './index';
+import {map} from 'rxjs/operator/map';
+
+
+describe('CdkTree', () => {
+  let fixture: ComponentFixture<SimpleCdkTreeApp>;
+
+  let component: SimpleCdkTreeApp;
+  let dataSource: FakeDataSource;
+  let tree: CdkTree<any>;
+  let treeElement: HTMLElement;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [CdkTreeModule],
+      declarations: [
+        SimpleCdkTreeApp,
+        // DynamicDataSourceCdkTreeApp,
+        // NodeContextCdkTreeApp,
+        // WhenNodeCdkTreeApp
```

Comment:
Add `TODO: ` in front of each of these? That will change them from commented-out-code to just comments